.github/workflows/connect-pre-release-init.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: yarn install
 
       - name: Build dependencies
-        # Running `build:libs` requires action to checkout with `submodules: recursive`.
+        # Running `build:libs` requires action to checkout with `submodules: true`.
         run: yarn build:libs
```

Comment:
We shouldn't run `build:libs` anymore, right? We should build directly what is needed.

Reply:
That's true you can now build any single package, but I guess we need to build all packages anyway because they are going to be published

Reply:
probably some of the build:libs can be removed from CI already, but there are still couple of problems so it can't be removed from everywhere.